libdrgn/python/symbol.c

```diff
@@ -36,6 +36,43 @@ static PyObject *Symbol_richcompare(Symbol *self, PyObject *other, int op)
 	Py_RETURN_BOOL(ret);
 }
 
+static PyObject *Symbol_source(PyObject *self, PyObject *args, PyObject* kwargs)
+{
+	char source[256];
+	char formatted[296];
```

Comment:
Where do the magic numbers come from? Comments would be nice here, though if you end up ditching the string formatting, then it will be irrelevant.